python/tvm/relay/op/tensor.py

```diff
@@ -63,6 +63,22 @@ def sqrt(data):
     return _make.sqrt(data)
 
 
+def rsqrt(data):
+    """Compute elementwise rsqrt of data.
```

Comment:
Could we add a mathematical description of the op which will be rendered in the docs?

Reply:
Sure